docker-compose.yml

```diff
@@ -64,6 +64,7 @@ services:
         environment:
           PDF_SERVICE_ENDPOINT: "http://pdfservice:9000/pdf"
           PDF_SERVICE_KEY: "pdfservice-api-key"
+          LOG_LEVEL: "DEBUG"
```

Comment:
It was mainly for diagnosing the issue, but I was thinking it might be useful to leave on by default. I could see it being a bit noisy, though. Do you think `INFO` would be a preferable default here?

Reply:
No, I think that's fine. For some reason I was thinking this would be applied in production, but as a default dev setting it seems reasonable.